runtime/ops/plugin.rs

```diff
@@ -43,111 +36,43 @@ pub fn op_open_plugin(
   debug!("Loading Plugin: {:#?}", filename);
   let plugin_lib = Library::open(filename).map(Rc::new)?;
   let plugin_resource = PluginResource::new(&plugin_lib);
+  mem::forget(plugin_lib);
```

Comment:
I guess this prevents the segfault when the process exits?

If so could you add a comment here.

Reply:
Correct this is to prevent segfaults, added a comment about it.